src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -145,7 +145,7 @@
 
     <!-- Default behavior is to disable OuterLoop and failing tests if not specified in WithCategories. -->
     <ItemGroup>
-      <DefaultNoCategories Include="OuterLoop" />
+      <DefaultNoCategories Condition="'$(Outerloop)'!='true'" Include="OuterLoop" />
```

Comment:
Should we generalize this to instead be something like /p:SkipDefaultCategories=true?  Then outer loop runs could use /p:SkipDefaultCategories=true /p:WithoutCategories=failing ?